data/categories.txt

```diff
@@ -1163,13 +1173,15 @@ shop-grocery|@food
 en:Grocery
 et:Toidukaubad
 fr:Épicerie
+hi:किराने की दुकान
 ru:Бакалея
 tr:Market
 
 shop-health_food|@food
 en:Health food
 de:4Reformhaus|5Naturkostladen|4Bioladen
 et:Tervisetoit
+hi:हेल्थ फ़ूड स्टोर
```

Comment:
Add "स्वस्थ भोजन की दुकान" as primary alias.

Reply:
there is no ranking in search words, as far as I know. Do you want your suggestion to be visible when you click on a point of interest?

Reply:
Yea